### PR TITLE
Add plus icon to project row on dashboard

### LINKED ISSUE #2866: Add plus icon to project row on dashboard
...to quickly create a goal in that project

### PR DESCRIPTION
**Pull request by IgorGoryany**
## PR includes

- [x] Feature

## Related issues

Resolve #2866

## QA Instructions, Screenshots, Recordings

Снимок экрана 2024-08-12 в 12 25 33

Снимок экрана 2024-08-12 в 12 25 18

### CODE REVIEW

**Comment by tsumo** [comment]
Tests are failing

File: src/components/StarButton/StarButton.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ interface IconProps {
     filled: boolean;
 }
 
-const Icon: React.FC<IconProps> = ({ filled }) => {
+export const Icon: React.FC<IconProps> = ({ filled }) => {
```
Comment (tsumo) [inline src/components/StarButton/StarButton.tsx:19]: StarButtonBase would be a better name for exported component

File: src/components/ProjectSubscriptionIconButtons/ProjectSubscriptionIconButtons.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    };
+
+    const onAddClick = (e: MouseEvent<HTMLButtonElement>) => {
+        e.preventDefault();
+        e.stopPropagation();
+        dispatchModalEvent(ModalEvent.GoalCreateModal, { project })();
+    };
+    return (
+        <>
+            <button className={s.ProjectSubscriptionIconButtons} onClick={onWatchClick} {...watch.attr}>
```
Comment (tsumo) [inline src/components/ProjectSubscriptionIconButtons/ProjectSubscriptionIconButtons.tsx:47]: Why not harmony buttons?

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/ProjectListItem/ProjectListItem.tsx [hunk 3]
```diff
@@ -62,25 +68,25 @@ export const ProjectListItem: React.FC<ProjectListItemProps & ComponentProps<typ
...
-                        />
-                    ),
+                {nullable(!editable, () => (
                     <>
                         {nullable(starred, () => (
                             <IconStarSolid size="s" />
                         ))}
                         {nullable(watching, () => (
                             <IconEyeOutline size="s" {...watch.attr} />
                         ))}
```
Comment (asabotovich) [inline src/components/ProjectListItem/ProjectListItem.tsx:78]: I think we don't need this anymore

It should be `<ProjectSubscriptionButtons view='icons'  editable={false} />`

**Review by asabotovich — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

**Pull request merged**